content/id/guides/configuration-glossary/configuration-ignore.md

```diff
@@ -0,0 +1,64 @@
+---
+title: 'Properti ignore'
+description: Tentukan berkas ignore untuk aplikasi Nuxt.js Anda
```

Comment:
```suggestion
description: Tentukan berkas yang diabaikan untuk aplikasi Nuxt.js Anda
```

Reply:
Menurut saya di sini konteksnya menyebutkan jenis file mas, yaitu file _ignore_. Karena di versi Bahasa Inggrisnya kata _ignore_ tidak menggunakan bentuk pasif (_Define the ignored files...._).

Reply:
Awalnya saya juga mengira demikian, tapi konteks kalimat bahasa Inggrisnya:

_Define the ignore files for your Nuxt.js application_

Kata `the ignore files`, karena ada artikel `the` jadi menurut saya itu mengacu pada `berkas-berkas yang diabaikan`. Bukan sebagai `berkas ignore`. Sehingga full terjemahannya kurang lebih:

_Tentukan berkas-berkas yang diabaikan untuk aplikasi Nuxt.js Anda_

Kalau ngga salah dokumentasi bagian itu menjelaskan kalau kita dapat memberi tahu Nuxt berkas-berkas mana saja yang diabaikan, diabaikan di sini tidak memicu _rebuild_ Nuxt.js ketika terjadi perubahan konten pada berkas tersebut. Saya rasa deskripsi tersebut sesuai dengan penjelasan kontennya.

Bagaimana menurut @febrihidayan ?